lucet-runtime/lucet-runtime-internals/src/region/mmap.rs

```diff
@@ -127,6 +131,7 @@ impl RegionInternal for MmapRegion {
             region,
         };
 
+        // TLC: Here is another early return from this function.  Hm.
```

Comment:
Once the `Alloc` is created, you don't have to worry anymore about early returns, since the `Drop` impl on `Alloc` will put the slot back on the freelist

Reply:
Thanks for this guidance!

Reply:
I added a bit of documentation about this in the code.  This way, Future Tanya won't get confused again.  😊